src/main/java/edu/iis/powp/command/CommandDrawLineToPosition.java

```diff
@@ -1,5 +1,18 @@
 package edu.iis.powp.command;
 
-public class CommandDrawLineToPosition {
+import edu.iis.client.plottermagic.IPlotter;
 
+public class CommandDrawLineToPosition implements PlotterCommand{
+
+    private int x;
+    private int y;
+
+    @Override public void execute(IPlotter iPlotter) {
```

Comment:
Name plotted would be better. 'I' prefix was used to indicate interface (it make sense in the context of types).